src/envo/envo-edit.owl

```diff
@@ -28119,6 +28142,155 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000032> <http://purl.obolibrar
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000038> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_00001995>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000038> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_00002008>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000038> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_03000021>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000038> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002353> <http://purl.obolibrary.org/obo/ENVO_03000020>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000039> (cryoconite hole)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "http://glaciers.pdx.edu/Projects/Antarctica/CryoconiteHoles/Cryo_main.html") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000039> "A vertical hole within an ice mass, formed by ice melt associated with local temperature increases, due to increased absorption of solar radiation by cryoconite deposits.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000116> <http://purl.obolibrary.org/obo/ENVO_03000039> "The general semantics of depressions are still to be worked out see  https://github.com/EnvironmentOntology/envo/issues/486")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000039> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_03000039> "This class doesn't refer to the site but instead the depression or hole and therefore the adjacency to the cryoconite itself.")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000039> "cryoconite hole"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000039> <http://purl.obolibrary.org/obo/ENVO_00000309>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000039> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_03000038>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000039> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002353> <http://purl.obolibrary.org/obo/ENVO_01000721>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000039> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002353> <http://purl.obolibrary.org/obo/ENVO_03000020>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000040> (melt pond)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Melt_pond") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000040> "A small water body, which forms via a melting processes occuring within a continuous mass of ice such a glacier or on sea ice.")
```

Comment:
> open melt pond def =
> 
> An open, atmosphere exposed, water body which forms as a result of localised melting taking place on the surface of a glacier or a mass of sea ice, contained in the open cavity formed by that melting.

Isn't much of this covered in the `melt pond` definition? No need to repeat material unless it's vital for understanding the def.

Also, rephrase to match the form: "A melt pond which ... "

Considering @jzrapp's input (which includes defs I've also seem before), perhaps create specific classes to accommodate all cases:

```
meltpond
- atmosphere-exposed meltpond
- meltpond channel (this would be a meltpond that is exposed to the atmosphere and the water column)
- ice-encased meltpond (totally sealed)

```

@jzrapp when you say:
> while "closed" refers to being fully closed

Does this include being closed to the atmosphere and marine water?

> closed melt pond def =
> 
> An enclosed, non-atmosphere exposed, water body which forms as a result of localised melting taking place within a glacier or a mass of sea ice, contained in the cavity formed by that melting.

As above, use the "A melt pond which ... " form (genus, differentiae):

```
Def: A melt pond which is encased in ice and thus not directly exposed to the atmosphere.

Comment: In practice, there may be some openings which are small relative to the surface footprint of the meltpond.
```


> Also I think I can finally use the located in axiom for closed melt ponds
> 
> 'located in' some 'ice mass'

Yes, I think this actually works here.

> I've created an issue to addres continuum between melt pond and intraglacial lakes #489

Thanks!

> finally before I submit this PR, I will make sure to add the envoPolar subset ID to terms which belong in polarOn

Great, thanks. I'll edit the Makefile to generate the new subset when your PR is in.